jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -377,7 +377,7 @@ def _auto_slider_range(self, event=None):
         # by taking abs, this will work for wavelength or frequency units.
         half_range = abs(x_max - x_min) / x_mid
         ndec = -np.log10(half_range)
-        if ndec > 0:
+        if ndec > 0 and not np.isinf(ndec):
```

Comment:
ran into a traceback when testing and clearing all data from the viewer (range goes to zero, ndec goes to infinity)